src/main/java/org/sasanlabs/service/vulnerability/commandInjection/CommandInjection.java

```diff
@@ -93,13 +93,16 @@ StringBuilder getResponseFromPingCommand(String ipAddress, boolean predict)
             @RequestParam(IP_ADDRESS) String ipAddress, RequestEntity<String> requestEntity)
             throws ServiceApplicationException, IOException {
 
-        Supplier<Boolean> condition = () ->StringUtils.isNotBlank(ipAddress)
+        Supplier<Boolean> condition =
+                () ->
+                        StringUtils.isNotBlank(ipAddress)
                                 && !SEMICOLON_SPACE_LOGICAL_AND_PATTERN
                                         .matcher(requestEntity.getUrl().toString())
                                         .find();
         return new ResponseEntity<GenericVulnerabilityResponseBean<String>>(
                 new GenericVulnerabilityResponseBean<String>(
-                        this.getResponseFromPingCommand(ipAddress, condition.get()).toString(), true),
+                        this.getResponseFromPingCommand(ipAddress, condition.get()).toString(),
```

Comment:
i am not sure if this change is required. just for "eq" matcher you are doing these changes. can you please write some argument matcher in test case ?